package/benchmarks/zdt/README.md

```diff
@@ -0,0 +1,66 @@
+---
+author: Optuna team
+title: The ZDT Problem Collection
+description: The ZDT Problem Collection (Zitzler et al. 2000) is a widely-used benchmark suite for multi-objective optimization. This package is a wrapper of the optproblems library.
+tags: [benchmark, multi-objective, ZDT, optproblems]
+optuna_versions: [4.1.0]
+license: MIT License
+---
+
+## Abstract
+
+This package provides a wrapper of the [optproblems](https://www.simonwessing.de/optproblems/doc/index.html) library's ZDT test suite, which consists of 6 kinds of benchmark problems. For the details of the benchmark problems, please take a look at the original paper (Zitzler et al., 2000) in the reference section.
+
+## APIs
+
+### class `Problem(function_id: int, **kwargs: Any)`
+
+- `function_id`: Function ID of the WFG problem in \[1, 9\].
+- `kwargs`: Arbitrary keyword arguments, please refer to [the optproblems documentation](https://www.simonwessing.de/optproblems/doc/zdt.html) for more details.
+
+#### Methods and Properties
+
+- `search_space`: Return the search space.
+  - Returns: `dict[str, optuna.distributions.BaseDistribution]`
+- `directions`: Return the optimization directions.
+  - Returns: `list[optuna.study.StudyDirection]`
+- `__call__(trial: optuna.Trial)`: Evaluate the objective function and return the objective value.
+  - Args:
+    - `trial`: Optuna trial object.
+  - Returns: `float`
+- `evaluate(params: dict[str, float])`: Evaluate the objective function and return the objective value.
```

Comment:
```suggestion
- `evaluate(params: dict[str, float])`: Evaluate the objective functions and return the objective values.
```

Reply:
`params: dict[str, float]` is not correct for ZDT5.